virttest/utils_misc.py

```diff
@@ -2107,16 +2107,17 @@ def get_cpu_vendor(cpu_info="", verbose=True):
 
 def get_support_machine_type(qemu_binary="/usr/libexec/qemu-kvm"):
     """
-    Get the machine type the host support,return a list of machine type
+    Get the machine type the host support,return a tuple (s, c),
+    s is machine type list, and c is system version list
     """
     o = process.system_output("%s -M ?" % qemu_binary).splitlines()
     s = []
     c = []
     for item in o[1:]:
-        machine_list = re.split(r"\s+", item, 1)
-        if machine_list[0] != "none":
+        if not ("alias" in item or "none" in item):
```

Comment:
It's better to make it optional.
```python
def get_support_machine_type(..., filter_alias=False):
```

Reply:
updated, could you help review again?  thanks!